sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1302,3 +1303,18 @@ def __init__(self, error=None, is_fatal=False, description=None, position=None):
         self.is_fatal = is_fatal
         self.description = description
         self.position = position
+
+class BlobTokenAudience(TokenAudience):
+    """Audiences available for Blobs."""
+
+    @classmethod
+    def get_service_account_audience(cls, storageAccountName: str) -> Self:
```

Comment:
Two comments here:
- Your param name `storageAccountName` looks very .NET like :). Should probably change that to `account_name`.
- I think this method should have a different name but not entirely sure what it should be. It's a classmethod so you would use it like `BlobTokenAudience.this_thing(account_name)` and since its Blob specific it can have a Blob specific name. Maybe just `blob_service_audience` or something?